formats/detailed_guide/publisher/details.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "body",
+    "first_published_at"
+  ],
+  "properties": {
+    "body": {
+      "type": "string"
+    },
+    "first_published_at": {
+      "type": "string"
+    },
+    "image": {
+      "$ref": "#/definitions/image"
+    },
+    "scheduled_publication": {
+      "type": "string"
+    },
+    "alternative_attachment_email": {
+      "type": "string"
+    },
+    "related_mainstream_url": {
+      "type": "string"
+    },
+    "related_mainstream_title": {
+      "type": "string"
+    },
+    "additional_related_mainstream_url": {
+      "type": "string"
+    },
+    "additional_related_mainstream_title": {
+      "type": "string"
+    },
+    "excluded_nations": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "alternative_scotland_url": {
+      "type": "string"
```

Comment:
I don't know how that looks like. What does this need instead of `"type": "string"`?

```
    "related_mainstream_url": {
      "type": "string"
    },
    "additional_related_mainstream_url": {
      "type": "string"
    },
    "alternative_scotland_url": {
      "type": "string"
    },
    "alternative_wales_url": {
      "type": "string"
    },
    "alternative_nothern_ireland_url": {
      "type": "string"
    },
```